tests/fsharp/core/printf/test.fsx

```diff
@@ -384,6 +384,71 @@ module CheckDisplayAttributes8 =
        override x.ToString() = "2"
 
     test "cenwoiwe8" (lazy(sprintf "%A" (Foo()))) "[1; 2]"
+
+// Check one returning two strings
+module CheckDisplayAttributes9 =
```

Comment:
@enricosada understood, I was just following the established convention in the file. I didn't feel it appropriate to change the naming convention in a file with a multitude of modules.